dev/squidex/__json/app.json

```diff
@@ -10,19 +11,27 @@
   "properties": {
     "clients": {
       "type": "object",
-      "additionalProperties": {}
+      "additionalProperties": {
+        "$ref": "#/definitions/AppClientModel"
```

Comment:
You may have an old version of `sq`

Reply:
@fampinheiro  You can let this one through. #156 Will update the backups to the current CLI version.